### PR TITLE
Pulling in Instant Articles changes from 0.8.x-release

### PR DESCRIPTION
**Pull request by spra85**

### CODE REVIEW

**Comment by mparent61** [comment]
LGTM

**Comment by MichaelButkovic** [comment]
@spra85 added the additional `dfp_position` targeting to this branch

**Comment by spra85** [comment]
👍 @MichaelButkovic

File: bulbs/contributions/email.py [hunk 1]
```diff
@@ -16,6 +16,33 @@
...
+class ContributorReport(object):
+    """User specific logic for emails."""
+
+    def __init__(self, contributor):
+        self.contributor = contributor
+
+        self._body = None
+
+    @property
+    def body(self):
```
Comment (spra85) [inline bulbs/contributions/email.py]: Shouldn't be in here??

**Pull request merged**